.babelrc

```diff
@@ -26,10 +26,7 @@
     "development": {
       "compact": false
     },
-    "e2e": {
-      "plugins": ["istanbul"]
-    },
-    "ci": {
```

Comment:
"For some reason" `NODE_ENV=e2e` currently in GitHub E2E. Should ideally be `test` but there was a complication.

Reply:
Got it, reverting that part then

Reply:
I'll follow up and try to have `NODE_ENV=test` in GitHub E2E. As it should be.